modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -27,9 +27,9 @@
 
 ## Issue Tracker Edit Existing Issue [Manual Testing]
 1. User can access the page if they fulfill all the following conditions:
-	* they have `issue_tracker_developer` or `issue_tracker_reporter` permission 
+	* they have `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` or ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission 
 	* they have `access_all_profiles` or are a member of the site of the issue or the site has no issue.
-2. Users who have `issue_tracker_reporter` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `issue_tracker_developer` permission can make all changes on all issues that they can view. 
+2. Users who have ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` permission can make all changes on all issues that they can view. 
```

Comment:
```suggestion
2. Users who have `Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` permission can make all changes on all issues that they can view. 
```

Reply:
don't judge me. My eyes caught that and I had to ask for a change. ;)

Reply:
damn!!